book/chapter-11/README.md

```diff
@@ -84,21 +84,21 @@ const Link = styled.a`
   color: black;
 `;
 
-<Link href='http://google.com'>Google</Link>
+<Link href='https://app.cupt.net/tcas/round3.php'>TCAS</Link>
 ```
 
-There is again a mechanism for extending classes. We may still use the `Link` component but change the text color like so:
+นี่คือรูปแบบทั่วไปที่เอาไว้ใช้สร้าง Class เรายังคงสามารถใช้ Component `Link` แต่เปลี่ยนสีตัวอักษรได้ดังนี้:
 
 ```js
 const AnotherLink = styled(Link)`
   color: blue;
 `;
 
-<AnotherLink href='http://facebook.com'>Facebook</AnotherLink>
+<AnotherLink href='http://www.mdes.go.th/'>Try to DoS me!</AnotherLink>
 ```
 
-By far for me styled-components are probably the most interesting approach for styling in React. It is quite easy to create components for everything and forget about the styling. If your company has the capacity to create a design system and building a product with it then this option is probably the most suitable one.
+ตามความคิดเห็นผมแล้ว styled-components เป็นวิธีการที่น่าสนใจที่สุดแล้วในการตกแต่งใน React ( คนแปลก็ชอบครับ ใครๆก็รัก styled-components ฮ่า ๆ ) มันง่ายมากในการสร้างทุก ๆ อย่างและลดความยุ่งยากในการ Styling ลง ถ้าบริษัทของคุณมีความเข้าใจและพร้อมในการทำระบบดีไซน์และและการทำระบบด้วย Styled-components นี่คงจะเป็นทางเลือกที่ดีที่สุดแล้ว
```

Comment:
I don't sure if we could add some self-opinion in the translation?

Reply:
Going to delete it ASAP.